crates/cubecl-linalg/src/matmul/components/batch/shared.rs

```diff
@@ -28,6 +29,10 @@ pub(crate) fn gmm_execute<MS: MatmulSpec, GMM: global::Matmul<MS>>(
         batch_rhs += tmp % rhs.shape(b) * rhs.stride(b);
     }
 
+    let lhs = VirtualTensor::<MS::EG>::new::<TensorInput<MS::EG, MS::Args>>(&lhs);
+    let rhs = VirtualTensor::<MS::EG>::new::<TensorInput<MS::EG, MS::Args>>(&rhs);
+    let out = VirtualTensor::<MS::EG, ReadWrite>::new::<TensorOutput<MS::EG, MS::Args>>(&mut out);
```

Comment:
Could be a good idea to implement a trait like the following to simplify the above.

```rust
pub trait IntoVirtual<E: Numeric, IO> {
    fn into_virtual(self) -> VirtualTensor<E, IO>;
} 
```